homeassistant/components/alarm_control_panel/device_trigger.py

```diff
@@ -21,6 +22,7 @@
     STATE_ALARM_ARMED_AWAY,
     STATE_ALARM_ARMED_HOME,
     STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_ARMED_VACATION,
     STATE_ALARM_ARMING,
     STATE_ALARM_DISARMED,
```

Comment:
Make sure to import `STATE_ALARM_PENDING` if you use it later in the code.
```suggestion
    STATE_ALARM_DISARMED,
    STATE_ALARM_PENDING,
```

Reply:
this isn't a valid state at the moment. Don't see why this should be added as this is identical to STATE_ALARM_ARMING